lineman/app/models/user_model.coffee

```diff
@@ -11,6 +11,9 @@ angular.module('loomioApp').factory 'UserModel', (BaseModel) ->
     setupViews: ->
       @membershipsView = @recordStore.memberships.belongingTo(userId: @id)
       @notificationsView = @recordStore.notifications.belongingTo(userId: @id)
+      @groupsView = @recordStore.groups.belongingTo(id: { $in: @groupIds() })
```

Comment:
Woo views.

Also, 'belongingTo' is a bit of a misnomer, since it'll make a view out of whatever params you pass to it, foreign key thingy or not.